packages/falcon-ecommerce-uikit/src/Query/Query.tsx

```diff
@@ -13,15 +13,39 @@ export class Query<TData = any, TVariables = OperationVariables, TTranslations =
     fetchMore?: (data: TData, fetchMore: QueryResult<TData, TVariables>['fetchMore']) => any;
     getTranslations?: (t: TranslationFunction, data: TData) => TTranslations;
     translationsNamespaces?: string[];
+    handleErrors?: boolean;
   }
 > {
+  getErrorCode(error: ApolloError | undefined): string | undefined {
```

Comment:
```suggestion
  getErrorCode(error?: ApolloError): string | undefined {
```

I'm not sure, but I believe we could define "optional" type with `?:`

Reply:
yes we can, it was simply taken from apollo d.ts